instances/events-committee.near/widget/devhub/components/molecule/SimpleMDE.jsx

```diff
@@ -139,7 +138,13 @@ function getSuggestedAccounts(term) {
   let results = [];
 
   term = (term || "").replace(/\W/g, "").toLowerCase();
-  const limit = 5;
+  let limit = 5;
+  if (term.length < 2) {
+   results = [${sortedRelevantUsers
+     .map((u) => "{accountId:'" + u + "', score: 60}")
+     .join(",")}];
+    limit = ${5 + sortedRelevantUsers.length};
+  }
```

Comment:
shouldn't we use reuse `SimpleMDE` from the devhub instance?

Reply:
Yes defenitly, we can do it in the next iteration of improvements.